BrainPortal/app/helpers/scope_helper.rb

```diff
@@ -681,13 +681,10 @@ def self.model_filter_values(model, attribute, label, association: nil)
     # Fetch the main filter values as an array of arrays:
     # [[value, label, count], [...]]
     model
-      .where("#{attribute} IS NOT NULL")
       .order(label, attribute)
       .group(attribute, label)
       .raw_rows(attribute, "#{label} AS #{label_alias}", "COUNT(#{attribute})")
       .reject { |r| r.first.blank? }
-      .map(&:to_a)
-      .to_a
```

Comment:
@remibernard I'm not sure why these lines were there... raw_rows() already returns an array of array, the reject() doesn't change that, so the last two lines don't do anything. The method still always return the same type as before, an array of arrays...